java/ql/src/Security/CWE/CWE-312/SensitiveStorage.qll

```diff
@@ -243,3 +250,111 @@ class Marshallable extends ClassStore {
     )
   }
 }
+
+/* Holds if the method call is a setter method of `SharedPreferences`. */
```

Comment:
Hi @smowton,

Most code can be easily moved to experimental as I did for query tests in the first version. However, currently I'm using the following code to integrate with <code>SensitiveSourceFlowConfig</code> in <code>SensitiveStorage.qll</code>.
```
    or
    exists(MethodAccess m |
      m.getMethod() instanceof SharedPreferences::SetPreferenceMethod and
      sink.asExpr() = m.getArgument(1)
    )
```

How could I do this when I move the rest of the query to experimental? Please advise. 

Thanks,
@luchua-bc

Reply:
Or shall I not to integrate at this point?

Reply:
I would say don't integrate yet -- if you pitch this at experimental without changing the results of existing queries first then it makes the review easier, because the bar for `experimental` is lower. Then you can either leave us to promote to `main` as and when it's high-priority for us, or make a separate push to promote the query out of experimental which won't hold up this PR, bounty payments and so on.

Reply:
Thanks @smowton for the confirmation. I've moved the query to experimental. The query structure is still pretty similar to existing queries, which facilitates future promotion when needed.

Reply:
👍  -- if you want even put comments about how the promotion should go

Reply:
I've added comments about how the future promotion should go.